libnetwork/osl/interface_linux.go

```diff
@@ -69,7 +95,13 @@ type Interface struct {
 	routes      []*net.IPNet
 	bridge      bool
 	sysctls     []string
-	ns          *Namespace
+	// advertiseAddrNMsgs is the number of unsolicited ARP/NA messages that will be sent to
+	// advertise the interface's addresses.
```

Comment:
Could you please mention that unsolicited ARP/NA are disabled when `advertiseAddrNmsgs` is set to zero?

Reply:
Done.